app/controllers/settings_controller.rb

```diff
@@ -9,11 +9,11 @@ class SettingsController < ApplicationController
   end
 
   def index
-    @settings = Setting.live_descendants.search_for(params[:search])
+    @settings = Setting.live_descendants.search_for(params[:search]).where(nil)
```

Comment:
Yeah, sorry. It was a remnant from some other PR I made to core that had to be compatible with both Rails 3 and 4. Thanks